cmd/oras/repo/tags.go

```diff
@@ -73,7 +71,7 @@ Example - Show tags associated with a digest:
 		},
 	}
 	cmd.Flags().StringVar(&opts.last, "last", "", "start after the tag specified by `last`")
-	cmd.Flags().BoolVar(&opts.excludeDigestTag, "exclude-digest-tags", false, "exclude all digest-like tags such as 'sha256-aaaa...'")
+	cmd.Flags().BoolVar(&opts.excludeDigestTag, "exclude-digest-tags", false, "[Preview] exclude all digest-like tags such as 'sha256-aaaa...'")
```

Comment:
Who does this that this option even exists?

Reply:
If referrers API is not available, an OCI registry might provide refer query capabilities via [tagged index](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api), thus `sha256-xxxxx` tags will be created. This flag can hide the generated tags.

Reply:
See
- #658 
- #668

for details.